src/components/ProxyGroup/ProxyEditItem/ProxyEditItem.js

```diff
@@ -108,12 +108,12 @@ class ProxyEditItem extends React.Component {
     let error = '';
 
     // proxy user expired
-    if (moment(proxyRel.user.expirationDate).endOf('day').isSameOrBefore(today)) {
+    if (proxyRel.user.expirationDate && moment(proxyRel.user.expirationDate).endOf('day').isSameOrBefore(today)) {
```

Comment:
This would fail I believe with the error _Cannot read property 'expirationDate' of undefined_ if `proxyRel` is `{}`. 

Might be better to use something like lodash `get` here:

`if( get(proxyRel, 'user.expirationDate') && ... )` 

which would result to `undefined` and fail the condition even if `proxyRel` is `{}`